sci-libs/rocThrust/rocThrust-5.1.3.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+ROCM_VERSION=${PV}
+
+inherit cmake rocm
+
+DESCRIPTION="HIP back-end for the parallel algorithm library Thrust"
+HOMEPAGE="https://github.com/ROCmSoftwarePlatform/rocThrust"
+SRC_URI="https://github.com/ROCmSoftwarePlatform/rocThrust/archive/rocm-${PV}.tar.gz -> rocThrust-${PV}.tar.gz"
+
+LICENSE="Apache-2.0"
+KEYWORDS="~amd64"
+SLOT="0/$(ver_cut 1-2)"
+IUSE="benchmark test"
+REQUIRED_USE="${ROCM_REQUIRED_USE}"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-util/hip
```

Comment:
```patch
@@ -13,13 +15,14 @@                                                                                      
 KEYWORDS="~amd64"       
 SLOT="0/$(ver_cut 1-2)"
 IUSE="benchmark test"
+REQUIRED_USE="${ROCM_REQUIRED_USE}"
                                                    
 RESTRICT="!test? ( test )"                                                                              
                                                    
-RDEPEND="dev-util/hip:=                                                                                 
-       sci-libs/rocPRIM:${SLOT}"                                                                        
-DEPEND="${RDEPEND}
+RDEPEND="dev-util/hip
+       sci-libs/rocPRIM:${SLOT}[${ROCM_USEDEP}]
        test? ( dev-cpp/gtest )"     
+DEPEND="${RDEPEND}"
 BDEPEND=">=dev-util/cmake-3.22"
             
 S="${WORKDIR}/rocThrust-rocm-${PV}"
```

Put in your commit message why there is no need to rebuild rocThrust if hip is upgraded.

Reply:
Done

Reply:
Also added similar information in rocBLAS/rocFFT commit